README.md

```diff
@@ -53,7 +53,7 @@ Many of the core components and extensions to VS Code live in their own reposito
 
 ## Bundled Extensions
 
-VS Code includes a set of built-in extensions located in the [extensions](extensions) folder, including grammars and snippets for many languages. Extensions that provide rich language support (code completion, Go to Definition) for a language have the suffix `language-features`. For example, the `json` extension provides coloring for `JSON` and the `json-language-features` extension provides rich language support for `JSON`.
+VS Code includes a set of built-in extensions located in the [extensions](extensions) folder, including grammar and snippets for many languages. Extensions that provide rich language support (code completion, Go to Definition) for a language have the suffix `language-features`. For example, the `json` extension provides the coloring for `JSON` and the `json-language-features` extension provides rich language support for `JSON`.
```

Comment:
Incorrect change of grammars to grammar. Plural is needed in this context.

Added 'the' is redundant.

Reply:
# Words in Phrase
![vscode-grammar-2](https://user-images.githubusercontent.com/69510006/158612280-df526de0-86cb-42f9-a3ab-343697d4f79e.JPG)
# Article
![vscode-grammar-3](https://user-images.githubusercontent.com/69510006/158612378-1f54105d-8721-46a7-bbfb-27cc961ecf7a.JPG)
# Determiner
![vscode-grammar-4](https://user-images.githubusercontent.com/69510006/158612408-631b55d8-6585-4246-b5ce-7ba5fa8d3311.JPG)